### DISCUSSION TITLE
Request GitHub SOC 2, Type 2 Report

### DISCUSSION THREAD

**Discussion by francisjames** [community/community#38605] [Other Feature Feedback, Questions, & Ideas]
We are required to provide our SOC 2 Type 2 Auditor the GitHub SOC 2 Type 2 Report.  I found where it states that it has moved under Compliance, but I get a 404 page not found when I use the link provided.
How do I obtain or download one?

**Comment by ossanna16** [comment]
Hi there @francisjames and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
